### DISCUSSION TITLE
Stepper driver connection

### DISCUSSION THREAD

**Discussion by oppozith** [kachurovskiy/megaels#26] [Q&A]
Hi.
Need help to understand correct connection in this case. Attached.

THX!

![20240407_011443.jpg](https://github.com/kachurovskiy/megaels/assets/5749538/4370a7f8-1f01-4059-bfab-2e80d009eff0)

![SmartSelect_20240407_011526_Chrome.jpg](https://github.com/kachurovskiy/megaels/assets/5749538/65444ba7-23b0-4fa9-b4b9-1e70853a52e4)

**Comment by kachurovskiy** [comment]
Hi @oppozith, have you tried https://github.com/kachurovskiy/nanoels/tree/main/h4#wiring ? 


    5V - connect to stepper driver PUL+, DIR+, ENA+ and wire shielding if there's any
    ENA - connect to stepper driver ENA-
    DIR - connect to stepper driver DIR-
    STEP - connect to stepper driver PUL-

**Reply by oppozith** [reply]
Yepp.
No lack. 
Seems that something wrong it other place ;)

**Reply by oppozith** [reply]
Found it!
Two broken patch cords :)))))
